NOTICE

```diff
@@ -10,6 +10,57 @@ Third party libraries used by the Beats project:
 
 
 --------------------------------------------------------------------
+github.com/Microsoft/go-winio
```

Comment:
I assume the change to the notice file is mainly to reordering. Perhaps we should find a way to sort the Notice entries in the future to not have duplicates and always the same sort order (alphabetical).